### PR TITLE
Farridav/upgrade to drf3

### PR DESCRIPTION
**Pull request by farridav**
Upgrade this package to support DRF3

- Convert serilaiser to field
- Ensure JSONField is returned as json

### CODE REVIEW

File: django_warnings/views.py [hunk 1]
```diff
@@ -17,12 +17,12 @@ class MyWarningViewSet(WarningViewSet):
     """
 
     model = Warning
```
Comment (ryankask) [inline django_warnings/views.py]: It seems like `ModelSerializer.model` is now redundant/not used. rockabox/rbx_campaigns#371 should probably be updated as well.

**Comment by ryankask** [comment]
nice: rbx_campaigns: `OK!  401 tests, 0 failures, 0 errors, 2 skips in 38.7s`

File: django_warnings/views.py (hunk 1 above)
Comment (ryankask) [inline django_warnings/views.py]: I've removed the `model` attribute in the `ViewSet`s in the `rbx_campaigns` PR.

Comment (farridav) [inline django_warnings/views.py]: ah ok, that explains a few things

**Comment by ryankask** [comment]
Looks good but the major version should be bumped so the rbx_campaigns requirements can be updated.

**Pull request merged**